_distutils_hack/__init__.py

```diff
@@ -1,6 +1,8 @@
 # don't import any costly modules
+from __future__ import annotations
 import sys
 import os
+from types import TracebackType
```

Comment:
> # don't import any costly modules

Here I don't have the choice to import either `from types import TracebackType` or `from typing import TYPE_CHECKING`. Dunno which one is "costlier"